src/views.rs

```diff
@@ -202,6 +202,7 @@ pub struct EncodableVersion {
     pub id: i32,
     #[serde(rename = "crate")]
     pub krate: String,
+    pub description: Option<String>,
```

Comment:
instead of adding this additional property on the JSON object that does not correspond to the database layout at all, I think it would be better to include the full `crate` resources in the response. the `version` resource already has a relationship to the `crate` in the line above this one. the only thing to look out for is to not do a N+1 database fetch when collecting the data for the response.